### ISSUE TITLE
[BUG] Actions not working for models with string ID, but only on Show page

### ISSUE THREAD

**Issue by aldent95** [avo-hq/avo#470]
### Describe the bug
When using models with string IDs, actions will work just fine if you select one or more of the model on an index page and use an action.

However, when on a show page, Actions will not work as the ID is not passed during the action request in the fields['resource_ids'] field.

### Steps to Reproduce
1. Create a model with a String ID
2. Create an Avo resource for that model
3. Create an Avo action for that resource
4. Create an instance of that model
5. Observe the action working fine on the index page when selecting the instance of the model
6. Observe the action not working when firing the action from the show page of that instance

### Expected behavior
Actions should work fine when firing from the show page, regardless of ID type for the model.

### Actual behavior
Action doesn't pick up the resource ID when the ID is a string when firing the action from a show page.

### System configuration
**Avo version**: 1.11.4

**Rails version**: 6.0.3.6

**Ruby version**: 2.7.1

### Additional context
I've trace what I think is the root cause of the issue to app/components/avo/views/resource_show_component.html.erb:3
```ruby
data-selected-resources="[<%= @resource.model.id %>]"
```
If the ID is a string, that bit of code produces an array that cannot be JSON parsed in app/packs/js/controllers/action_controller:27

Changing the above code to 
```ruby
data-selected-resources='["<%= @resource.model.id %>"]'
```
resolves the issue for me, but I'm not sure what other effects that might have.

**Comment by adrianthedev** [comment]
Yeah. Not very happy about holding state in the DOM like that. Would probably sit better in `localStorage` somewhere.
That fix you pointed out would be good for now though.

**Comment by luonet** [comment]
The same bug for me when using action with string ID in show page.
The resources_id in posted form data is empty.

**Comment by adrianthedev** [comment]
Released in [1.11.6](https://rubygems.org/gems/avo/versions/1.11.6)
